runtime/compiler/control/CompilationThread.hpp

```diff
@@ -1,4 +1,4 @@
-/*******************************************************************************
```

Comment:
May I ask you to add these three asterisks back in again?  There was no need to delete them.